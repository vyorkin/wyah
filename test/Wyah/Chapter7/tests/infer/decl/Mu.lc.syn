Decl "Mu" (ELam (Var "f") (EApp (EVar (Var "f")) (EFix (EVar (Var "f")))))
