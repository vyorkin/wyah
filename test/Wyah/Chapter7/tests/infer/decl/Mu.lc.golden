Forall [TV "a"] ((TVar (TV "a") :-> TVar (TV "a")) :-> TVar (TV "a"))